editor/plugins/path_2d_editor_plugin.cpp

```diff
@@ -179,6 +178,41 @@ bool Path2DEditor::forward_gui_input(const Ref<InputEvent> &p_event) {
 			return true;
 		}
 
+		// Check for segment split.
+		if (mb->is_pressed() && mb->get_button_index() == BUTTON_LEFT && mode == MODE_EDIT && on_edge == true) {
+			Vector2 gpoint = mb->get_position();
+			Ref<Curve2D> curve = node->get_curve();
+
+			int insertionPoint = -1;
```

Comment:
Still some `camelCase` identifiers in this code block and the next one, otherwise looks good.

Reply:
Good grief.  Habit is hard to overcome.